grpc/tls/provider/meshca/v1/config.proto

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grpc.tls.provider.meshca.v1;
```

Comment:
Hmm, actually, the proto package here probably warrants some discussion.  If this same config proto is going to be used in Envoy, does it make sense for its proto package to be in the `grpc` namespace?

Reply:
Options that come to my mind:
`google.tls.provider.meshca.v1`
`tls.provider.google.meshca.v1`
`certificate.provider.google.meshca.v1`

The good thing about #2 and #3 is that we could have `tls.provider.vault.*` etc.

Reply:
The problem is that I don't think we can lay claim to any of those top-level namespaces.

This is probably another reason why we should have a central authoritative place for this proto.  It should be in the namespace of whoever maintains the proto.  We'll have to discuss this when we meet.

I am okay with proceeding with this PR for now, as long as we agree that whatever code is written using the current namespace will be changed once we decide what we actually want to use here.  And since the current namespace is temporary, we should probably put "experimental" somewhere in the name.

Reply:
Would `grpc.tls.provider.meshca.experimental` work?

Reply:
Yeah, I guess that's fine for now.  Although at this point, the meeting is tomorrow, so we might as well wait to have a real decision before we proceed.

Reply:
Makes sense. Will hold off on this.

Reply:
Done.